src/map_reads.cpp

```diff
@@ -24,20 +24,21 @@
 ## -------------------------------------------------------------------------
 
 ## Authors (alphabetically): Jacob L., Jaillard M., Lima L.
-## Modified by John Lees
+## Modified by John Lees and Santeri Puranen
 */
 
 #include "global.h"
 #include "map_reads.hpp"
 #include "Utils.h"
 #include <boost/algorithm/string/predicate.hpp>
+#include <boost/dynamic_bitset.hpp>
 #include <map>
```

Comment:
Is this still used?

Reply:
Still using std::map, so you should keep it.